main.go

```diff
@@ -168,13 +167,6 @@ func main() {
 		ghOrg = "redhat-appstudio-appdata"
 	}
 
-	// Retrieve the name of the default repository to use
```

Comment:
Pls update the README to reflect this ie; it needs to be removed

Reply:
done, please check